README.md

```diff
@@ -1,3 +1,86 @@
+
+To import only the token manger, you can use this line: `import tokenManager = require('ibm-cloud-sdk-core/iam-token-manager/v1');`.
+
+### isFileParam
+This function takes an Object and returns `true` if the object is a Stream, a Buffer, has a `value` property, or has a `data` property that is a file param (checked recursively).
+
+### isEmptyObject
+This function takes an Object and returns `true` if it is an empty object.
+
+### getContentType
+This function take a ReadableStream, Buffer, or String and returns a String. It attempts to retrieve the content type of the input and returns `null` if not found.
+
+### stripTrailingSlash
+This function takes a String and returns an identical string but with a forward slash removed from the end, if present.
+
+### getMissingParams
+This function takes two arguments: an Object containing the method parameters and a string Array containing the list of required parameter names. It returns `null` if all required parameters are present and an Error if any are missing.
+
+### isHTML
+This function takes a String and returns `true` is the given string is HTML.
+
+### getFormat
+This function takes two arguments: an Object containing the method parameters and a string Array containing the list of formats. It returns the first match from the "formats" array that is a key in the parameters object, or `null` if there are no matches.
+
+### buildRequestFileObject
+This function takes a `FileParamAttributes` object and returns a `FileObject`. It builds a "form-data" object for each file parameter.
+
+#### FileParamAttributes
+**data**: NodeJS.ReadableStream | Buffer | FileObject
+**contentType**: string
+**filename**: string
+
+#### FileObject
+**value**: NodeJS.ReadableStream | Buffer | string
+**options?**: FileOptions
+
+#### FileOptions
+**filename?**: string
+**contentType?**: string
+
+### toLowerKeys
+This function takes an Object and returns the same object with all of the top-level keys converted to lower case. Note: it does not convert nested keys.
+
+### qs
+This module includes one function, `stringify`.
+
+#### qs.stringify
+This functions takes an Object containing query parameters and returns a URI-encoded query String. This function is modified for use with Watson - the query parameter `watson-token` will not be encoded, as the server requires non-encoded tokens for authentication.
+
+### contentType
+This module includes two functions, `fromHeader` and `fromFilename`.
+
+#### contentType.fromHeader
+This function takes a Buffer and returns a String. It reads the first bytes of the buffer and attempts to return the mime type associated with the file. Returns `undefined` if no matching mime type is found. Supported types: `audio/flac`, `audio/wav`, `audio/ogg`, `audio/mp3`, `audio/webm`
```

Comment:
where would the user obtain this "Buffer" from exactly?   this detail might help folks to understand how/if to use this function.